### PR TITLE
Test on latest version of solr

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -48,14 +48,21 @@ services:
...
+      - $PWD/lib/generators/spotlight/templates/solr/conf:/opt/solr/conf
+    ports:
+      - "${SOLR_PORT}:8983"
+    entrypoint:
+      - docker-entrypoint.sh
+      - solr-precreate
+      - blacklight-core
+      - /opt/solr/conf
+      - "-Xms256m"
+      - "-Xmx512m"
```
Comment (thatbudakguy) [inline docker-compose.yml:65]: why the JVM memory settings? I usually don't bother in docker unless something is breaking

**Review by thatbudakguy — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (jcoyne) [reply docker-compose.yml:65]: I just copied from blacklight: https://github.com/projectblacklight/blacklight/blob/main/docker-compose.yml#L33-L39  Want me to remove this?

Comment (thatbudakguy) [reply docker-compose.yml:65]: yeah, I think unless we need it better not to set anything.

Comment (jcoyne) [reply docker-compose.yml]: Okay, they've been removed.

Comment (thatbudakguy) [reply docker-compose.yml]: thank you!

**Pull request merged**